lib/saml_idp/signature_builder.rb

```diff
@@ -21,7 +22,8 @@ def raw
     end
 
     def x509_certificate
-      SamlIdp.config.x509_certificate
+      certificate = @x509_certificate || SamlIdp.config.x509_certificate
+      certificate
```

Comment:
Thoughts on putting the dot at the end of each line for consistency? As in:
```ruby
certificate.
to_s.
gsub
```

Reply:
Yeah, I guess rubocop didn't catch it b/c it's outside the diff.